Library/Homebrew/utils/github.rb

```diff
@@ -666,4 +666,37 @@ def pull_request_labels(user, repo, pr)
     pr_data = API.open_rest(url_to("repos", user, repo, "pulls", pr))
     pr_data["labels"].map { |label| label["name"] }
   end
+
+  def last_commit(user, repo, ref)
+    return if Homebrew::EnvConfig.no_github_api?
+
+    output, _, status = curl_output(
+      "--silent", "--head", "--location",
+      "--header", "Accept: application/vnd.github.sha",
+      "https://api.github.com/repos/#{user}/#{repo}/commits/#{ref}"
```

Comment:
```suggestion
      url_to("repos", user, repo, "commits", ref)
```

(might need a `.to_s` - not sure)